chocolatey/lektor.nuspec

```diff
@@ -0,0 +1,34 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>lektor</id>
+    <version>3.2.0</version>
+    <packageSourceUrl>https://github.com/lektor/lektor</packageSourceUrl>
+    <owners>Nimish Bongale</owners>
+
+    <title>lektor</title>
+    <authors>lektor</authors>
+    <projectUrl>https://www.getlektor.com/</projectUrl>
+    <!--<iconUrl>http://cdn.rawgit.com/__REPLACE_YOUR_REPO__/master/icons/lektor.png</iconUrl>-->
```

Comment:
This should use the Egg on feet logo: https://www.getlektor.com/logo.png (just the egg part I guess)

Reply:
Cool! I'll add the logo.